Services/Administration/OJS.Services.Administration.Business/Implementations/HangfireBackgroundJobsBusinessService.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace OJS.Services.Administration.Business.Implementations
+{
+    using System;
+    using System.Threading.Tasks;
+    using OJS.Services.Common;
+
+    public class HangfireBackgroundJobsBusinessService : IHangfireBackgroundJobsBusinessService
+    {
+        private readonly ISubmissionsForProcessingBusinessService submissionsForProcessing;
+
+        public HangfireBackgroundJobsBusinessService(ISubmissionsForProcessingBusinessService submissionsForProcessing)
+            => this.submissionsForProcessing = submissionsForProcessing;
+
+        public int EnqueuePendingSubmissionsJob() => this.submissionsForProcessing.EnqueuePendingSubmissions();
+
+        public void DeleteProcessedSubmissionsJob() => throw new NotImplementedException();
```

Comment:
This is not implemented anywhere

Reply:
This is a mistake, It should be implemented here. I'll fix it.